docs/development/accessibility-process.md

```diff
@@ -0,0 +1,283 @@
+# Accessibility Process
+
+The ILIAS community aims to provide a system that is usable for everyone, including
+users with special needs. This document clarifies how we want to move our software
+towards this goal incrementally, who is involved, which tools the community uses and
+how different activities are incorporated in our release cycle.
+
+**Table of Contents**
+* [Goals](#goals)
+* [Participants](#participants)
+* [Tools](#tools)
+* [Activities in Release Cycle](#activities)
+* [Future Development](#future-development)
+
+<a name="goals"></a>
+## Goals
+
+While the general goal is to provide a system that is usable to for everyone, that
+goal needs to be elaborated to some more concrete goals to become effective. These
+concrete goals are:
+
```

Comment:
ILIAS complies with the WACG Guidelines.

Reply:
ILIAS aims to comply with the WCA Guidelines. This process attempts to ensure constant improvements towards this goal.